api/src/list/list.service.ts

```diff
@@ -17,21 +14,25 @@ interface Comment {
   username: string;
 }
 
+export interface UpdateListData {
+  name?: string;
+  isPrivate?: boolean;
+}
```

Comment:
This should probably be a DTO.

Reply:
how could I, the king of DTOs, make such a grave error